plugins/inputs/jolokia2/jolokia_test.go

```diff
@@ -6,11 +6,12 @@ import (
 	"net/http/httptest"
 	"testing"
 
+	"github.com/stretchr/testify/assert"
```

Comment:
It would be best to revert these import sorting changes since they aren't related to the pull request.